### PR TITLE
8305622: Remove Permission details from jcmd man page

### PR DESCRIPTION
**Pull request by kevinjwalls**
Removal of the Permission information from the jcmd man page.

This information is not useful in the way it is presented, and is not used in connection with jcmd, so should be removed from that man page.

The Permissions can be relevant when connecting remotely to JMX with a Security Manager in place.  That usage is clearly deprecated with JEP 411.  If there is a transition feature, it will be documented elsewhere.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8305622](https://bugs.openjdk.org/browse/JDK-8305622): Remove Permission details from jcmd man page

### Reviewers
 * [Chris Plummer](https://openjdk.org/census#cjplummer) (@plummercj - **Reviewer**)
 * [Serguei Spitsyn](https://openjdk.org/census#sspitsyn) (@sspitsyn - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13363/head:pull/13363` \
`$ git checkout pull/13363`

Update a local copy of the PR: \
`$ git checkout pull/13363` \
`$ git pull https://git.openjdk.org/jdk.git pull/13363/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13363`

View PR using the GUI difftool: \
`$ git pr show -t 13363`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13363.diff">https://git.openjdk.org/jdk/pull/13363.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13363#issuecomment-1498167011)

### CODE REVIEW

**Comment by plummercj** [comment]
What about the permissions references in the jcmd help output:

```
$ jcmd 32228 help VM.version
32228:
VM.version
Print JVM version information.

Impact: Low

Permission: java.util.PropertyPermission(java.vm.version, read)

Syntax: VM.version
```

**Comment by kevinjwalls** [comment]
> What about the permissions references in the jcmd help output:
> 
> ```
> $ jcmd 32228 help VM.version
> 32228:
> VM.version
> Print JVM version information.
> 
> Impact: Low
> 
> Permission: java.util.PropertyPermission(java.vm.version, read)
> 
> Syntax: VM.version
> ```

Aha, ok that is created programmatically by HelpDCmd::execute... And I don't think we should print it, it has the same problems as the man page.  As it's a code change not a man page change, I think I should handle that in a separate CR which I'll log now...

**Comment by plummercj** [comment]
I ran jconsole, selected MBeans -> com.sun.management -> DiagnosticCommand -> Operations, and it shows the list of DCMDs (what we normally see as jcmds). I clicked on one and it has a `dcmd.permissionName` field with the permission in it. Is this an example of using JMX where you feel showing the permission is the right thing to do.

**Comment by kevinjwalls** [comment]
> 

Yes - JConsole (using DiagnosticCommandImpl.java)  showing the permissions is more relevant, I wasn't going to change that right now.

Removing the Permissions from the jcmd help output (in a separate change) will make that jconsole info better, as the Permission info is duplicated in the help text and the permission fields that it shows.

File: src/jdk.jcmd/share/man/jcmd.1 [hunk 1]
```diff
@@ -402,12 +378,6 @@ On Linux operating systems, the temporary directory is \f[V]/tmp\f[R].
 On Windwows, the temporary directory is specified by the \f[V]TMP\f[R]
 environment variable.)
 .IP \[bu] 2
-\f[V]preserve-repository=\f[R]{\f[V]true\f[R]|\f[V]false\f[R]} :
-Specifies whether files stored in the disk repository should be kept
-after the JVM has exited.
-If false, files are deleted.
-By default, this parameter is disabled.
-.IP \[bu] 2
```
Comment (plummercj) [inline src/jdk.jcmd/share/man/jcmd.1:410]: Why is this being removed?

Comment (dholmes-ora) [reply src/jdk.jcmd/share/man/jcmd.1:410]: I'm guessing the src used to generate the manpage was missing the very recent change.

**Comment by AlanBateman** [comment]
The SM permissions were interesting when connected to a MBeanServerConnection to access MXBeans remotely. A bit puzzling why they would be printed as the usage for commands executed by jcmd.

File: src/jdk.jcmd/share/man/jcmd.1 (hunk 1 above)
Comment (kevinjwalls) [reply src/jdk.jcmd/share/man/jcmd.1]: Yes, it was rush hour in the jcmd man page, the changes overlapped.  Updated this change.

**Review by plummercj — APPROVED** [review]

**Review by sspitsyn — APPROVED** [review]

**Comment by kevinjwalls** [comment]
Thanks for the reviews!

**Pull request closed without merging**